lib/opensusebasetest.pm

```diff
@@ -1164,6 +1164,7 @@ under test, the version and if the SUT is an upgrade.
 sub firewall {
     my $old_product_versions      = is_sle('<15') || is_leap('<15.0');
     my $upgrade_from_susefirewall = is_upgrade && get_var('HDD_1') =~ /\b(1[123]|42)[\.-]/;
+    return 'firewalld' if (check_var('SUSEFIREWALL2_SERVICE_CHECK', 1));
     return (($old_product_versions || $upgrade_from_susefirewall) && !is_tumbleweed) ? 'SuSEfirewall2' : 'firewalld';
```

Comment:
How do you think set the condition here? 

($old_product_versions || $upgrade_from_susefirewall) && !is_tumbleweed && ! (check_var('SUSEFIREWALL2_SERVICE_CHECK', 1))

Reply:
cool, will change  the code and rerun to verify it. thanks

Reply:
https://openqa.nue.suse.com/tests/5341421#step/firewall_enabled/1 passed